### ISSUE TITLE
Gradient norm: nan

### ISSUE THREAD

**Issue by ValerioNeriGit** [CannyLab/tsne-cuda#87]
Hi, 

I have a dataset of around  55.000 1024-D vectors, I'm getting nan gradients when increasing the num_neighbors option, above 50.
Is there something else I should change? A the moment I'm using:
`result = TSNE(n_components=n_components, verbose=3, num_neighbors=50, n_iter=5000).fit_transform(data)` 

I am on archlinux, on a docker container with cuda 10.2 installed via conda following your [issue](https://github.com/CannyLab/tsne-cuda/issues/81)

Thank you

**Comment by ValerioNeriGit** [comment]
I figured out I was not passing a float
